Tests/Tests/RouterSpec.swift

```diff
@@ -139,6 +177,11 @@ class RouterSpec: QuickSpec {
                     expect(window.findIAMView()).toEventually(beNil())
                 }
 
+                it("will not remove the campaign that doesn't exist") {
```

Comment:
I'm not sure I understand the context here. You try to call `discardDisplayedCampaign()` when no campaign is being displayed?

Reply:
Yes, to cover the failure case on `guard` check.

Reply:
Ok. I think to make it more sensible please rename this test to "will not crash when there are no displayed campaigns" AND remove `expect(window.findIAMView()).toEventually(beNil())` line. (Leaving only `router.discardDisplayedCampaign()`)